pkg/cmd/releasenotes/options.go

```diff
@@ -39,32 +39,38 @@ func NewGitGenerateOptions(streams genericclioptions.IOStreams) *GenerateOptions
 	}
 }
 
-func (o GenerateOptions) Validate() error {
+func (o *GenerateOptions) Validate() error {
 	var errs []error
+
 	if o.ReleaseName == "" {
 		errs = append(errs, fmt.Errorf("release name can't be empty"))
 	}
+
 	if strings.HasPrefix(o.ReleaseName, "v") {
 		errs = append(errs, fmt.Errorf("release name can't has 'v' prefix"))
 	}
+
 	if o.StartRef == "" {
 		errs = append(errs, fmt.Errorf("start ref can't be empty"))
 	}
+
 	if o.EndRef == "" {
 		errs = append(errs, fmt.Errorf("end ref can't be empty"))
 	}
 
+	if len(o.GithubToken) == 0 {
+		errs = append(errs, fmt.Errorf("github-token can't be empty"))
+	}
+
 	return errors.NewAggregate(errs)
 }
 
 func (o *GenerateOptions) Complete(ctx context.Context) error {
 	var httpClient *http.Client
```

Comment:
nit: without the condition this var could be initialized inline in L73

Reply:
done